### DISCUSSION TITLE
Add support to edit/change issue type

### DISCUSSION THREAD

**Discussion by sebastian-sus** [ankitpokhrel/jira-cli#469] [Ideas]
Currently, it's possible to set the type of an issue during creation (e.g. -tStory, -tBug etc).

It would be really great if it would be also possible to change the type after creation. On the web interface, it's always a multipage dialog and painful to do so manually.

I searched the discussions and issues but couldn't find such a request yet. I also tried to change the issue type via custom properties but that doesn't work either.

**Comment by riotrah** [comment]
Same! how can we help move this forward?
